client/components/TestQueue/Conflicts/index.jsx

```diff
@@ -0,0 +1,230 @@
+        testPlanVersion.updatedAt,
+        'YY.MM.DD'
+      )}`,
+      testTitle: conflict.conflictingResults[0].test.title,
+      testRowNumber: conflict.conflictingResults[0].test.rowNumber,
+      testSequenceNumber: getConflictTestNumberFilteredByAt(conflict),
+      testRenderedUrl: conflict.conflictingResults[0].test.renderedUrl,
+      atName: data.testPlanReport.at.name,
+      browserName: data.testPlanReport.browser.name,
+      atVersionName: data.testPlanReport.requiredAtVersion?.name
+    });
+  };
+
+  const { isAdmin } = useMemo(() => evaluateAuth(data?.me), [data?.me]);
+
+  const disclosureLabels = useMemo(() => {
+    return data?.testPlanReport?.conflicts?.map(conflict => {
+      const testIndex = getConflictTestNumberFilteredByAt(conflict);
+      return `Test ${testIndex}: ${conflict.conflictingResults[0].test.title}`;
+    });
+  }, [data?.testPlanReport?.conflicts]);
+
+  const disclosureContents = useMemo(() => {
+    return data?.testPlanReport?.conflicts?.map(conflict => {
+      const issues = data?.testPlanReport?.issues?.filter(
+        issue =>
+          issue.testNumberFilteredByAt ===
+          getConflictTestNumberFilteredByAt(conflict)
+      );
+      return (
+        <ConflictSummaryTable
+          issues={issues}
+          issueLink={getIssueLink(conflict)}
+          isAdmin={isAdmin}
+          key={conflict.conflictingResults[0].test.id}
+          conflictingResults={conflict.conflictingResults}
+          testIndex={getConflictTestNumberFilteredByAt(conflict)}
+        />
+      );
+    });
+  }, [data?.testPlanReport?.conflicts]);
+
+  const disclosureClickHandlers = useMemo(() => {
+    return data?.testPlanReport?.conflicts?.map((_, index) => () => {
+      setOpenDisclosures(prevState => {
+        const newOpenDisclosures = [...prevState];
+        newOpenDisclosures[index] = !newOpenDisclosures[index];
+        return newOpenDisclosures;
+      });
+    });
+  }, [data?.testPlanReport?.conflicts]);
+
+  if (error) {
+    return (
+      <PageStatus
+        title="Conflicts | ARIA-AT"
+        heading="Conflicts"
+        message={error.message}
+        isError
+      />
+    );
+  }
+
+  if (loading) {
+    return (
+      <PageStatus title="Loading - Conflicts | ARIA-AT" heading="Conflicts" />
+    );
+  }
+
+  const {
+    title,
+    versionString,
+    id: testPlanVersionId
+  } = data?.testPlanReport.testPlanVersion ?? {};
+  const { name: browserName } = data?.testPlanReport.browser ?? {};
+  const { name: atName } = data?.testPlanReport.at ?? {};
+  const { name: requiredAtVersionName } =
+    data?.testPlanReport.requiredAtVersion ?? {};
+  const { name: minimumAtVersionName } =
+    data?.testPlanReport.minimumAtVersion ?? {};
+
+  return (
+    <PageContainer id="main" as="main" tabIndex="-1">
+      <Helmet>
+        <title>
+          Conflicts {title} {versionString} | ARIA-AT
+        </title>
+      </Helmet>
+      <h1>Conflicts for Test Plan Report</h1>
```

Comment:
```suggestion
      <h1>Conflicts for Test Plan Report {title} {versionString}</h1>
```

Maybe also the AT and Browser for further specificity? But we can see what others think here as well since it is already listed in the meta details below.

Reply:
Yeah I went back and forth on this. At first, I had the title, version string, at, minimum or required at version, and browser all in the title but that felt too busy. Then I added the meta details section to free up some complexity in the title. This could be a good middle point though. Changes here [0d029ce](https://github.com/w3c/aria-at-app/pull/1195/commits/0d029ce756f2ec944eb1d4c514b51ccb2dbe8bb8)